create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -114,15 +116,16 @@ void shouldLogExceptionMessageWhenExceptionOccurs() {
     }
 
     @Test
-    void shouldExtractScopusIdentifierAndPlaceItInsideAdditionalIdentifiersObject() throws IOException {
-        var scopusFile = IoUtils.stringFromResources(Path.of(SCOPUS_XML_0000469852));
-        var uri = s3Driver.insertFile(UnixPath.of(randomString()), scopusFile);
+    void shouldExtractOnlyScopusIdentifierIgnoreAllOtherIdentifiersAndStoreItInPublication() throws IOException {
+        var scopusData = new ScopusGenerator();
+        var scopusIdentifiers = keepOnlyTheScopusIdentifiers(scopusData);
+        var uri = s3Driver.insertFile(UnixPath.of(randomString()), scopusData.toXml());
         var s3Event = createS3Event(uri);
         var createPublicationRequest = scopusHandler.handleRequest(s3Event, CONTEXT);
         var actualAdditionalIdentifiers = createPublicationRequest.getAdditionalIdentifiers();
-        var expectedIdentifier =
```

Comment:
In the previous code, we have an inconsistency between the test and the code, which is actually visible in the test.
According to the code we may have more than one scopus identifiers for the same object and we will store them all. 
But here we are testing only the most common case, which is one identifier. 

By introducing the random data generator, we can generate the actual case and test that we will store all scopus identifiers. Note, that at this point we are not interested in whether expecting one or more scopus identifiers is correct or not. We are only interested in expressing what has been implemented.